OpenRA.Mods.Common/Traits/World/MPStartLocations.cs

```diff
@@ -130,7 +130,7 @@ void INotifyCreated.Created(Actor self)
 			spawnLocations = spawns.ToArray();
 
 			// Initialize the list of unoccupied spawn points for AssignSpawnLocations to pick from
-			availableSpawnPoints = Enumerable.Range(1, spawnLocations.Length).ToList();
+			availableSpawnPoints = Enumerable.Range(1, spawnLocations.Length).Except(self.World.LobbyInfo.DisabledSpawnPoints).ToList();
```

Comment:
> I rebased on bleed and surprisingly there were no conflicts, which was a first. 👍

Not quite!

It wasn't an explicit conflict, but we will now also need to make a matching change in `object IAssignSpawnPointsInfo.InitializeState(MapPreview map, Session lobbyInfo)` in `MPStartLocationsInfo` above.